core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -352,6 +353,12 @@ public void run() {
     @Override public boolean isReady() {
       return false;
     }
+
+    @Override
+    public void setDecompressor(Decompressor d) {}
```

Comment:
Message decoding is almost always done with a string rather than a proper Decompressor.  However, to mirror the setCompressor, I added this method.  This is in internal, so I wasn't trying too hard to keep this clean.  If you think there should only be one method, I can remove one of them.